chapters/capabilities.adoc

```diff
@@ -568,6 +576,11 @@ endif::VK_NV_external_memory_sci_buf[]
 ifdef::VK_QNX_external_memory_screen_buffer[]
 | ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_SCREEN_BUFFER_BIT_QNX | No restriction | No restriction
 endif::VK_QNX_external_memory_screen_buffer[]
+ifdef::VK_EXT_external_memory_metal[]
+| ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_MTLBUFFER_BIT_EXT | Must match | Must match
+| ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_MTLTEXTURE_BIT_EXT | Must match | Must match
+| ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_MTLHEAP_BIT_EXT | Must match | Must match
+endif::VK_EXT_external_memory_metal[]
```

Comment:
I'm concerned about the choice of "Must match" in this table. I suspect that the constraint is impossible to satisfy in the extension's intended use cases.

When sharing a Metal handle with a different VkDevice in a different VkInstance, the "must match" constraint is easy to satisfy. That's not my concern.

I assume that an intended use case for the extension is to share Metal handles *outside* of Vulkan. (Please correct me if I'm wrong). In that case, do the relevant sharing apis expose the equivalent of `driverUUID`, and does that UUID match the Vulkan driver's `driverUUID`?

Reply:
> I assume that an intended use case for the extension is to share Metal handles outside of Vulkan.

For the Android Emulator, sharing outside of the Vulkan is not a priority, as we won't be sharing the handle with OpenGL on Mac, and we share the buffers and images only between different Vulkan instances and devices.

Reply:
Maybe changing `driverUUID` here to `No restriction` should be the way forward since Metal does not have an equivalent. The only requirement should be that the `deviceUUID` matches, and Metal does provide an equivalent.

Reply:
Based on today's discussion in the SI meeting, this proposal sgtm. No requirement on driverUUID, but require matching deviceUUID.